code/clue.js

```diff
@@ -7,53 +7,332 @@ const mrGreen = {
 const pickKiller = () => {
+  // Tells the animation to not be shown until card is clicked 
+  document.getElementById('loader-wrapper-killer').style.display = "none";
+  document.getElementById('killer-card-wrapper').style.display = "block";
+
   // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+  mystery.killer = randomSelector(suspects);
 
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerCard').style.background = '#e7cb96'; //mystery.killer.color; (commented out because it looks awful)
+  let killerImage = document.getElementById('killerImage');
+  killerImage.src = mystery.killer.image;
+  //document.getElementById('killerName').style.background = '#900d0d'; 
+  document.getElementById('killerName').style.padding = '15px';
+  //document.getElementById('killerName').style.margin = '0px';
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`;
+  document.getElementById('killerOccupation').innerHTML
+  document.getElementById('killerAge').innerHTML = `${mystery.killer.occupation}, ${mystery.killer.age}`;
+  document.getElementById('killerDescription').innerHTML = `${mystery.killer.description}`;
+  shuffleFavouriteWeapon();
+};
+console.log(pickKiller);
+
+const loadingPickKiller = () => {
+  //Decides animation timer
+  const loaderTime = setTimeout(pickKiller, 1500);
+  //Hides the cards first side 
+  document.getElementById("start-display-card-killer").style.display = "none";
+  //Hides the card information when first when clicked (because the animation will be shown first)
+  document.getElementById('killer-card-wrapper').style.display = "none";
+  //Displays the animation
+  document.getElementById('loader-wrapper-killer').style.display = "flex";
+  document.getElementById('killerCard').style.background = '#900d0d';
 }
 
+// Telling the function to start when card onclicks
+document.getElementById("killerCard").onclick = loadingPickKiller;
+
 // CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
+const pickWeapon = () => {
+  // Tells the animation to not be shown until card is clicked
+  document.getElementById('loader-wrapper-weapon').style.display = "none";
+  document.getElementById('weapon-card-wrapper').style.display = "block";
+  // This will randomly select a weapon from the weapons. And add that to the mystery object.
+  mystery.weapons = randomSelector(weapons);
+
+  // This will change styling 
+  document.getElementById('weaponCard').style.background = '#e7cb96';
```

Comment:
All of those style you add via JS should be done by CSS. In JS, you should only add/remove/toggle CSS classes